wrapper/src/main/java/software/amazon/jdbc/plugin/efm/MonitorImpl.java

```diff
@@ -25,11 +25,16 @@
 import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.concurrent.atomic.AtomicLong;
 import java.util.logging.Logger;
+
 import org.checkerframework.checker.nullness.qual.NonNull;
+
```

Comment:
Why are there extra blank lines in the import list? These lines would fail the checkstyle.

Reply:
Fixed